tests/test_activities.py

```diff
@@ -658,3 +658,35 @@ def test_activity2_download_as_pdf_async(self):
             pdf_file_called_after_activity = os.path.join(target_dir, activity_name + '.pdf')
             self.assertTrue(pdf_file)
             self.assertTrue(pdf_file_called_after_activity)
+
+    @skip
+    def test_activity2_share_link(self):
+        activity_name = 'Task - Form + Tables + Service'
+        message = 'EXAMPLE_MESSAGE'
+        subject = 'EXAMPLE_SUBJECT'
+        recipient_users = [96]
+
+        activity = self.project.activity(name=activity_name)
```

Comment:
send it to no-reply+pykechaintesting@ke-chain.com

Reply:
Done